.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,6 +1,9 @@
 ---
 name: Bug report
 about: File a bug report
+title: ''
+labels: ''
```

Comment:
@jevansaks should we have this automatically label bugs?

Reply:
No, because bugs are sometimes documentation or spec -- I'd like to let triage decide.